runtime/pattern.go

```diff
@@ -204,6 +217,15 @@ func (p Pattern) Match(components []string, verb string) (map[string]string, err
 	return bindings, nil
 }
 
+// Match examines components if it matches to the Pattern.
+// If it matches, the function returns a mapping from field paths to their captured values.
+// If otherwise, the function returns an error.
+//
+// Deprecated: Use MatchAndEscape.
```

Comment:
I know this was the existing comment, but lets reword it.
```suggestion
// Match examines components to determine if it matches to the Pattern.
// If it matches, the function returns a mapping from field paths to their captured values.
// Otherwise, the function returns an error.
//
// Deprecated: Use MatchAndEscape.
```

Reply:
Happy to, how about:

```
// MatchAndEscape examines components to determine if they match to a Pattern. It will
// never perform path unescaping (UnescapeModeLegacy). MatchAndEscape will return an error
// if no Patterns matched. If successful, the function returns a mapping from field paths to their 
// captured values.
//
// Deprecated: Use MatchAndEscape.
```